java/java-impl/src/inspectionDescriptions/RedundantCompareToJavaTime.html

```diff
@@ -0,0 +1,19 @@
+<!-- Copyright 2000-2022 JetBrains s.r.o. and contributors. Use of this source code is governed by the Apache 2.0 license. -->
+<html>
+<body>
+Reports comparisons in which the <code>compareTo</code> method can be replaced to <code>isAfter</code>, <code>isBefore</code> or <code>isEqual</code>
+of LocalTime, LocalDateTime, LocalDate, OffsetTime or OffsetDateTime.
```

Comment:
I would replace with something like:
```
Reports <code>java.time</code> comparisons with <code>compareTo()</code> calls that can be replaced with 
<code>isAfter()</code>, <code>isBefore()</code> or <code>isEqual()</code> calls.
```
motivations:
- this mentions java.time immediately to avoid confusion about which kind of compareTo() methods it handles
- all references to methods include  parentheses to avoid confusion
- mentioning the individual classes becomes unnecessary because of the java.time package reference.

Reply:
This phrase is much better. Changed. Thank you!